benchmarks/single_node/test_single_node.py

```diff
@@ -14,6 +16,16 @@
 MAX_RAY_GET_SIZE = 100 * 2**30
 
 
+def assert_no_leaks():
+    total = ray.cluster_resources()
+    current = ray.available_resources()
+    total.pop("memory")
+    total.pop("object_store_memory")
+    current.pop("memory")
+    current.pop("object_store_memory")
+    assert total == current, (total, current)
```

Comment:
so this is to ensure no CPU resource leakage?

Reply:
yup